hack/lib/golang.sh

```diff
@@ -558,6 +560,17 @@ kube::golang::build_binaries() {
     local use_go_build
     local -a targets=()
     local arg
+    
+    # Add any files with those //generate annotations in the array below.
+    readonly BINDATAS=( "${KUBE_ROOT}/test/e2e/framework/gobindata_util.go" )
+    for bindata in ${BINDATAS[@]}; do
+	    # Only try to generate bindata if the file exists, since in some cases
+	    # one-off builds of individual directories may exclude some files.   
+            if [[ -f $bindata ]]; then
+		go generate "${bindata}"
```

Comment:
yeah, its pretty fast < 1 sec , but worth optimizing in the future if lots of builds are being done